src/sliding-tab/ExNavigationSlidingTab.js

```diff
@@ -42,6 +42,7 @@ type Props = {
   position: "top" | "bottom",
   pressColor: ?string,
   renderBefore: () => ?ReactElement<any>,
+  renderTabBar: (props: Object) => React.Element<{}>,
```

Comment:
Since you're using `renderTabBar` as a component, why restrict the signature to a function? Should also allow class components and React.createClass too.

Personally I'd prefer to call the function directly instead of using it as a component to keep it consistent with other things.